### PR TITLE
feat(ingestion/Vertica): Fixed vertica integration test Updated vertica dialect

### PR DESCRIPTION
**Pull request by vishalkSimplify**
## Checklist

- [x] The PR conforms to DataHub's [Contributing Guideline](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md) (particularly [Commit Message Format](https://github.com/datahub-project/datahub/blob/master/docs/CONTRIBUTING.md#commit-message-format))
- [x] Links to related issues (if applicable)
- [x] Tests for the changes have been added/updated (if applicable)
- [x] Docs related to the changes have been added/updated (if applicable). If a new feature has been added a Usage Guide has been added for the same.
- [x] For any breaking change/potential downtime/deprecation/big changes an entry has been made in [Updating DataHub](https://github.com/datahub-project/datahub/blob/master/docs/how/updating-datahub.md)

1. Modified Vertica Integration test and updated Vertica database version in integration test.
2. Updated Vertica logo.
3. Updated Vertica dialect to the latest version.

### CODE REVIEW

File: metadata-ingestion/tests/integration/vertica/test_vertica.py [hunk 1]
```diff
@@ -37,28 +35,24 @@ def vertica_runner(docker_compose_runner, test_resources_dir):
...
-        assert ret.returncode >= 1
+        assert ret.returncode == 0
 
         yield docker_services
 
 
 # Test needs more work to be done , currently it is working fine.
 @freeze_time(FROZEN_TIME)
-@pytest.mark.skip("Failing in CI, cmd failing with exit code 1")
+# @pytest.mark.skip("Failing in CI, cmd failing with exit code 1")
```
Comment (hsheth2) [inline metadata-ingestion/tests/integration/vertica/test_vertica.py:55]: just remove this line instead

File: metadata-ingestion/tests/integration/vertica/test_vertica.py [hunk 2]
```diff
@@ -72,10 +66,11 @@ def test_vertica_ingest_with_db(vertica_runner, pytestconfig, tmp_path):
     ignore_paths: List[str] = [
         r"root\[\d+\]\['proposedSnapshot'\].+\['aspects'\].+\['customProperties'\]\['create_time'\]",
         r"root\[\d+\]\['proposedSnapshot'\].+\['aspects'\].+\['customProperties'\]\['table_size'\]",
-        r"root\[\d+\]\['proposedSnapshot'\].+\['aspects'\].+\['customProperties'\]\['projection_size'\]",
+        r"root\[\d+\]\['proposedSnapshot'\].+\['aspects'\].+\['customProperties'\]\['Projection_size'\]",
         r"root\[\d+\]\['proposedSnapshot'\].+\['aspects'\].+\['customProperties'\]\['ROS_Count'\]",
         r"root\[\d+\]\['aspect'\].+\['customProperties'\]\['cluster_size'\]",
         r"root\[\d+\]\['aspect'\].+\['customProperties'\]\['udx_language'\]",
+        r"root\[\d+\]\['aspect'\].+\['json'\]\['rowCount'\]",
```
Comment (hsheth2) [inline metadata-ingestion/tests/integration/vertica/test_vertica.py:73]: why would the row count change between runs? shouldn't this be consistent on every run of CI?

Comment (vishalkSimplify) [reply metadata-ingestion/tests/integration/vertica/test_vertica.py:73]: The line was added during the debugging of the test. I have removed it now. Thanks for pointing it out.

File: metadata-ingestion/tests/integration/vertica/test_vertica.py (hunk 1 above)
Comment (vishalkSimplify) [reply metadata-ingestion/tests/integration/vertica/test_vertica.py:55]: sure . removed it

**Review by hsheth2 — APPROVED** [review]

**Pull request merged**